packages/core/src/hub.ts

```diff
@@ -66,6 +74,7 @@ export interface Layer {
 export interface Carrier {
   __SENTRY__?: {
     hub?: Hub;
+    asyncContextStrategy?: AsyncContextStrategy;
```

Comment:
can we name this something more bundle size efficient 😄

the type should be descriptive enough.